Frontend/ui-common/src/Common/Constants/Constant.purs

```diff
@@ -0,0 +1,32 @@
+
+module Common.Constants where
```

Comment:
@S-PRAVINKUMAR we already have one file for constant in ui-common, as `Contants.purs` can we add this value there instead of creating a new file.